website/docs/amazon-emr-on-eks/emr-eks-karpenter.md

```diff
@@ -2,32 +2,158 @@
+| CloudWatch metrics | No | CloudWatch container insights metrics shows simple and standardized way to monitor not only AWS resources but also EKS resources on CloudWatch dashboard | [CloudWatch Container Insights Documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Container-Insights-metrics-EKS.html) |
+|AWS for Fluent-bit | No | This can be used to publish EKS cluster and worker node logs to CloudWatch Logs or 3rd party logging system | [AWS For Fluent-bit Documentation](https://github.com/aws/aws-for-fluent-bit) |
+| FSx for Lustre CSI driver | No | This can be used for running Spark application using FSx for Lustre | [FSx for Lustre CSI Driver Documentation](https://docs.aws.amazon.com/eks/latest/userguide/fsx-csi.html) |
+
+
+### Customizing Add-ons
+
+You can customize your deployment anytime either by changing recommended system add-ons in `addons.tf` or by changing optional add-ons in `variables.tf`. 
+
+For example, let's say you want to remove Amazon Managed Prometheus because you have another application that captures Prometheus metrics, you can edit `addons.tf` using your favorite editor, find Amazon Managed Prometheus and change to `false`
+```
+enable_amazon_prometheus             = false
+```
+
+If you want to use FSx for Lustre storage while running Spark application for storing shuffle files or accessing data from S3, you can install FSx CSI driver by searching for FSx in `variables.tf` and edit the file
+```
+variable "enable_aws_fsx_csi_driver" {
+  default     = true
+```
+Once the changes are saved, follow the [deployment guide](#deploying-the-solution) if this is a new installation or apply these changes using Terraform for existing installation
+```
+terraform apply
+```
+
+## Deploying the Solution
+
+Let's go through the deployment steps 
+
+### Prerequisites:
+
+Ensure that you have installed the following tools on your machine.
+
+1. [aws cli](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html)
+2. [kubectl](https://Kubernetes.io/docs/tasks/tools/)
+3. [terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli)
+
+_Note: Currently Amazon Managed Prometheus supported only in selected regions. Please see this [userguide](https://docs.aws.amazon.com/prometheus/latest/userguide/what-is-Amazon-Managed-Service-Prometheus.html) for supported regions._
+
+First, clone the repository
+
+```bash
+git clone https://github.com/awslabs/data-on-eks.git
+```
+
+Navigate into one of the example directories and run `terraform init`
+
+```bash
+cd data-on-eks/analytics/terraform/emr-eks-karpenter
+terraform init
+```
+
+Set AWS_REGION and Run Terraform plan to verify the resources created by this execution.
+
+```bash
+export AWS_REGION="us-west-2"
+terraform plan
+```
+
+This command may take between 20 and 30 minutes to create all the resources.
+
+```bash
+terraform apply
```

Comment:
I have also created a `install.sh` script. We can mention to users that they can use it if they prefer

Reply:
sure